.github/workflows/labels.yml

```
@@ -0,0 +1,44 @@
+name: Add Label
+
+on:
+  pull_request:
+    types: [ edited, labeled, opened, reopened ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  no_ticket:
+    name: "Label whether ticket linked from title"
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: kaisugi/action-regex-match@v1.0.1
+        id: regex-match
+        name: "Check title"
+        with:
+          text: ${{ github.event.pull_request.title }}
+          regex: '#[0-9]+ '
```

Some contributors forget to use the `#`:
```suggestion
          regex: '[0-9]{6} '
```

---

Based off https://github.com/django/code.djangoproject.com/blob/72f85efc886e5c5ddfceb1b4a3793af247873c02/trac-env/htdocs/tickethacks.js#L39 from Trac, tickets are linked to Trac if they contain:
- `#<ticket_id> `
- `#<ticket_id>,`
- `#<ticket_id>:`
- `#<ticket_id>` 

So it must have a `#` to be linked to Trac, I think that some people forget is fine because anyone can update the title

---

Makes sense, thank you for the detailed explanation!